storage/libradb/src/lib.rs

```diff
@@ -193,11 +202,31 @@ impl LibraDB {
         error_if_too_many_requested(limit, MAX_LIMIT)?;
 
         let get_latest = !ascending && start_seq_num == u64::max_value();
+        let account_state =
+            self.get_account_state_with_proof(query_path.address, ledger_version, ledger_version)?;
+        let account_resource = if let Some(account_blob) = &account_state.blob {
+            AccountResource::make_from(&(&account_blob.try_into()?))?
+        } else {
+            bail!("Nothing stored under access path: {}", query_path.address);
+        };
+        let access_path = if query_path.path == account_sent_event_path() {
```

Comment:
Maybe it's a little bit more clear to call it `event_handle`? (Even if we don't rename the Type for now.)